Makefile

```diff
@@ -74,14 +74,16 @@ curl_local curl_prod:
 	--data "@./data/sample_deal.json" \
 	${CURL_URL}
 
-logfile := logs/src.api.$(shell date +%FT%T).log
-pidfile := logs/server.PID
+LOGDIR  := logs/
+logfile := ${LOGDIR}src.api.$(shell date +%FT%T).log
+pidfile := ${LOGDIR}server.PID
 
 start_local_server: libdds-build stop_local_server
 	@echo
-	@echo Starting a local test service as a background process. You can stop it with \`make stop_local_service\`
+	@echo Starting a local test service as a background process. You can stop it with \`make stop_local_server\`
 	@echo The server log will be written to \'${logfile}\'.
 	@echo
+	mkdir ${LOGDIR} &> /dev/null || true
```

Comment:
`mkdir -p` is a better choice when creating directories because it doesn't fail if directory exists.

But I would still use makefile mechanism to avoid running mkdir when directory exists:
```
${TMPDIRS} := \
       ${LOGDIR} \
# end of TMPDIRS

${TMPDIRS}:
    mkdir -p $@
```

Then the target creates the directory when it doesn't exists. It can create any other directory which is added to  TMPDIRS variable.
I uses a list idiom which allows later additions and removals to modify only affected lines. The diff with only relevant lines is easier to read when reviewing changes.

As I wonder if there is any need to create other directories it could be simpler just write:
```
${LOGDIR}:
    mkdir -p $@
```

Reply:
I like. Done.